CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Changelog
 
-## UNRELEASED
+# v0.112.0
```

Comment:
Can we make it v0.111.1 and not make the glue catalog exporter public just yet (no proper tests/docs in place)